nengo/utils/tests/test_progress.py

```diff
@@ -1,9 +1,50 @@
 import time
 
 import pytest
+import numpy as np
 
+import nengo.rc as rc
 from nengo.exceptions import ValidationError
-from nengo.utils.progress import AutoProgressBar, Progress, ProgressBar
+from nengo.utils.progress import (
+    AutoProgressBar,
+    Progress,
+    ProgressBar,
+    _load_class,
+    WriteProgressToFile,
+    get_default_progressbar,
+)
+
+
+def test_load_class():
+    """tests _load_class"""
+    name = "numpy.sin"
+    _load_class(name) == np.sin
+
+
+def test_write_progress_to_file():
+    """tests WriteProgressToFile"""
+    filename = "test_progress_file"
+    prog = WriteProgressToFile(filename)
+
+    class Test:
+        def __init__(self, isfinished):
+            self.finished = isfinished
+
+        def myFunction(self):
+            return 1
+
+        progress = 0
+        name_after = "newname"
+        elapsed_seconds = myFunction
+
+        def eta(self):
+            return 0
+
+    data = Test(False)
+    prog.update(data)
+
+    data = Test(True)
+    prog.update(data)
```

Comment:
What does this test assert? We should probably open the written file and make sure it contains what we expect.